QR code Genrator for Employee/requirment.txt

```diff
@@ -0,0 +1,4 @@
+If you run this project in your machine then you have to required 
```

Comment:
```suggestion

```
Instead of writing the modules manually use the following command `pip freeze > requirement.txt`.